TeamCode/src/main/java/org/firstinspires/ftc/teamcode/helpers/utils/MotionProfile.java

```diff
@@ -0,0 +1,194 @@
+package org.firstinspires.ftc.teamcode.helpers.utils;
+
+import com.acmerobotics.dashboard.FtcDashboard;
+import com.acmerobotics.dashboard.telemetry.MultipleTelemetry;
+import com.arcrobotics.ftclib.controller.PIDController;
+import com.qualcomm.robotcore.util.ElapsedTime;
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+
+public class MotionProfile {
```

Comment:
add a comment here specifying what this motion profile does exactly, like [here](https://github.com/Vilnius-Lyceum-Robotics/into-the-deep/blob/f5d20fd877c37a0641e89cca1e7f2b27a39543e8/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/controls/PrimaryDriverTeleOpControls.java#L11)

i would also add comments to all of the methods, but this isnt really neccesary